src/reynir_correct/errtokenizer.py

```diff
@@ -803,6 +817,44 @@ def description(self) -> str:
         return self._txt
 
 
+@register_error_class
+class RitmyndirError(Error):
+    """A RitmyndirError is a context-independent error from Ritmyndir data"""
+
+    # Miscellaneous error codes, detailed here: https://bin.arnastofnun.is/gogn/storasnid/ritmyndir/
+
+    def __init__(
+        self,
+        code: str,
+        txt: str,
+        detail: Optional[str],
+        references: Optional[List[str]],
+        original: str,
+        suggest: str,
+    ) -> None:
+        super().__init__(code, original=original, suggest=suggest)
+        self._txt = txt
+        self._detail = detail
+        self._references = references
+
+    @property
+    def description(self) -> str:
+        return self._txt
+
+    @property
+    def detail(self) -> Optional[str]:
+        return self._detail
+
+    @property
+    def references(self) -> Optional[List[str]]:
+        return self._references
+
+    def to_dict(self) -> Dict[str, Any]:
+        d = super().to_dict()
+        d["detail"] = self.detail
```

Comment:
`self._detail` is slightly more appropriate here

Reply:
Done